.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,89 @@
+---
+name: docker_image
+
+# https://docs.github.com/en/actions/publishing-packages/publishing-docker-images
+# https://docs.docker.com/build/ci/github-actions/multi-platform
+# https://github.com/opencontainers/image-spec/blob/main/annotations.md
+
+on:
+  push:
+    branches: [ "master" ]
+    # Publish semver tags as releases.
+    tags: [ "*.*.*" ]
```

Comment:
What happens if you have both `branches` and `tags` in a workflow? Is it `and` or `or`?

Reply:
Ah, given that the docs mention latest master, it must be "or". Looks correct, resolving